2020-06-01T10:52:15.637695+00:00 app[web.1]: npm ERR! Exit status 1
2020-06-01T10:52:15.637857+00:00 app[web.1]: npm ERR! 
2020-06-01T10:52:15.638006+00:00 app[web.1]: npm ERR! Failed at the eiazweihaushaltshilfe@1.0.0 start script.
2020-06-01T10:52:15.638110+00:00 app[web.1]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2020-06-01T10:52:15.648000+00:00 app[web.1]: 
2020-06-01T10:52:15.648381+00:00 app[web.1]: npm ERR! A complete log of this run can be found in:
2020-06-01T10:52:15.648880+00:00 app[web.1]: npm ERR!     /app/.npm/_logs/2020-06-01T10_52_15_639Z-debug.log
2020-06-01T10:52:15.719352+00:00 heroku[web.1]: Process exited with status 1
2020-06-01T10:52:15.757752+00:00 heroku[web.1]: State changed from up to crashed
2020-06-01T10:52:20.874179+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=agkeia.herokuapp.com request_id=665bba23-e4ff-4dd2-b275-51ddefa4e4c7 fwd="46.5.229.154" dyno=web.1 connect=5000ms service= status=503 bytes= protocol=https
2020-06-01T10:54:25.978975+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/?product=Kekse&Kekse=1&product=Schokolade&Schokolade=1&product=Brot&Brot=1&market=Aldi" host=agkeia.herokuapp.com request_id=2e5c57a6-5b8f-40e5-9c0e-c0aca42f9368 fwd="46.5.229.154" dyno= connect= service= status=503 bytes= protocol=https
2020-06-01T10:57:34.932202+00:00 heroku[web.1]: State changed from crashed to starting
2020-06-01T10:57:38.169704+00:00 heroku[web.1]: Starting process with command `npm start`
2020-06-01T10:57:40.375642+00:00 app[web.1]: 
2020-06-01T10:57:40.375659+00:00 app[web.1]: > eiazweihaushaltshilfe@1.0.0 start /app
2020-06-01T10:57:40.375660+00:00 app[web.1]: > node L07_Database/Server/L07_Server.js
2020-06-01T10:57:40.375660+00:00 app[web.1]: 
2020-06-01T10:57:40.951902+00:00 heroku[web.1]: State changed from starting to up
2020-06-01T10:57:40.732983+00:00 app[web.1]: Server starting on port:53402
2020-06-01T10:58:10.818537+00:00 app[web.1]: (node:23) UnhandledPromiseRejectionWarning: MongoServerSelectionError: connection <monitor> to 3.127.163.16:27017 closed
2020-06-01T10:58:10.818582+00:00 app[web.1]:     at Timeout._onTimeout (/app/node_modules/mongodb/lib/core/sdam/topology.js:430:30)
2020-06-01T10:58:10.818762+00:00 app[web.1]:     at listOnTimeout (internal/timers.js:549:17)
2020-06-01T10:58:10.818792+00:00 app[web.1]:     at processTimers (internal/timers.js:492:7)
2020-06-01T10:58:10.818809+00:00 app[web.1]: (node:23) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
2020-06-01T10:58:10.818811+00:00 app[web.1]: (node:23) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
2020-06-01T10:59:20.800247+00:00 app[web.1]: What's up?
2020-06-01T10:59:20.804337+00:00 app[web.1]: /app/L07_Database/Server/L07_Server.js:44
2020-06-01T10:59:20.804338+00:00 app[web.1]:         orders.insert(_order);
2020-06-01T10:59:20.804339+00:00 app[web.1]:                ^
2020-06-01T10:59:20.804339+00:00 app[web.1]: 
2020-06-01T10:59:20.804341+00:00 app[web.1]: TypeError: Cannot read property 'insert' of undefined
2020-06-01T10:59:20.804344+00:00 app[web.1]:     at storeOrder (/app/L07_Database/Server/L07_Server.js:44:16)
2020-06-01T10:59:20.804344+00:00 app[web.1]:     at Server.handleRequest (/app/L07_Database/Server/L07_Server.js:39:13)
2020-06-01T10:59:20.804345+00:00 app[web.1]:     at Server.emit (events.js:315:20)
2020-06-01T10:59:20.804345+00:00 app[web.1]:     at parserOnIncoming (_http_server.js:790:12)
2020-06-01T10:59:20.804346+00:00 app[web.1]:     at HTTPParser.parserOnHeadersComplete (_http_common.js:119:17)
2020-06-01T10:59:20.814574+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2020-06-01T10:59:20.815054+00:00 app[web.1]: npm ERR! errno 1
2020-06-01T10:59:20.816316+00:00 app[web.1]: npm ERR! eiazweihaushaltshilfe@1.0.0 start: `node L07_Database/Server/L07_Server.js`
2020-06-01T10:59:20.816466+00:00 app[web.1]: npm ERR! Exit status 1
2020-06-01T10:59:20.816641+00:00 app[web.1]: npm ERR! 
2020-06-01T10:59:20.816788+00:00 app[web.1]: npm ERR! Failed at the eiazweihaushaltshilfe@1.0.0 start script.
2020-06-01T10:59:20.817023+00:00 app[web.1]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2020-06-01T10:59:20.826560+00:00 app[web.1]: 
2020-06-01T10:59:20.826722+00:00 app[web.1]: npm ERR! A complete log of this run can be found in:
2020-06-01T10:59:20.826834+00:00 app[web.1]: npm ERR!     /app/.npm/_logs/2020-06-01T10_59_20_819Z-debug.log
2020-06-01T10:59:20.882377+00:00 heroku[web.1]: Process exited with status 1
2020-06-01T10:59:20.926828+00:00 heroku[web.1]: State changed from up to crashed
2020-06-01T10:59:20.811442+00:00 heroku[router]: at=error code=H13 desc="Connection closed without response" method=GET path="/?product=Kekse&Kekse=1&product=Schokolade&Schokolade=1&product=Eis&Eis=1&market=Aldi&money=Geld+einzahlen&bargeld=30&household=Gassi+gehen&household=Medikamente+besorgen" host=agkeia.herokuapp.com request_id=f2f79fff-1add-49bf-b19d-3ed2463981f4 fwd="46.5.229.154" dyno=web.1 connect=0ms service=13ms status=503 bytes=0 protocol=https
2020-06-01T11:08:34.585624+00:00 heroku[web.1]: State changed from crashed to starting
2020-06-01T11:08:38.186198+00:00 heroku[web.1]: Starting process with command `npm start`
2020-06-01T11:08:41.296365+00:00 app[web.1]: 
2020-06-01T11:08:41.296388+00:00 app[web.1]: > eiazweihaushaltshilfe@1.0.0 start /app
2020-06-01T11:08:41.296389+00:00 app[web.1]: > node L07_Database/Server/L07_Server.js
2020-06-01T11:08:41.296389+00:00 app[web.1]: 
2020-06-01T11:08:41.603537+00:00 app[web.1]: Server starting on port:26700
2020-06-01T11:08:42.519867+00:00 heroku[web.1]: State changed from starting to up
2020-06-01T11:09:11.709720+00:00 app[web.1]: (node:23) UnhandledPromiseRejectionWarning: MongoServerSelectionError: connection <monitor> to 18.157.173.36:27017 closed
2020-06-01T11:09:11.709740+00:00 app[web.1]:     at Timeout._onTimeout (/app/node_modules/mongodb/lib/core/sdam/topology.js:430:30)
2020-06-01T11:09:11.709742+00:00 app[web.1]:     at listOnTimeout (internal/timers.js:549:17)
2020-06-01T11:09:11.709743+00:00 app[web.1]:     at processTimers (internal/timers.js:492:7)
2020-06-01T11:09:11.710014+00:00 app[web.1]: (node:23) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
2020-06-01T11:09:11.710145+00:00 app[web.1]: (node:23) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.